packages/xpetra/CMakeLists.txt

```diff
@@ -149,12 +120,23 @@ IF (${PACKAGE_NAME}_ENABLE_Epetra)
           MESSAGE(FATAL_ERROR "If Xpetra Epetra support is enabled and Epetra 32-bit global indices are enabled, Xpetra requires that you enable Tpetra_INST_INT_INT")
         ELSE()
           # Epetra was enabled but only implicitly; give a warning but turn off Xpetra's Epetra support
-          MESSAGE(WARNING "32-bit global indices are enabled in Epetra but not Tpetra. Since Epetra was not enabled explicitly by user, disabling Epetra support in Xpetra.")
-          SET(${PACKAGE_NAME}_ENABLE_Epetra OFF)
+          MESSAGE(WARNING "32-bit global indices are enabled in Epetra but not Tpetra. Since Epetra was not enabled explicitly by user, **DISABLING EPETRA SUPPORT** in Xpetra and its downstream."
```

Comment:
"and its downstream" ==>  "and its downstream dependencies"

Reply:
Thanks, I made this message clearer.